sdk/cosmos/azure-cosmos-examples/src/main/java/com/azure/cosmos/examples/BasicDemo.java

```diff
@@ -20,7 +20,7 @@
 
     private static final String DATABASE_NAME = "test_db";
     private static final String CONTAINER_NAME = "test_container";
-
+    int count = 0;
```

Comment:
it does not look like we are using this "count".

Reply:
Removed